README.md

```diff
@@ -37,23 +37,32 @@ TBD
 ### Usage
 
 ```javascript
-let a = new Intl.RelativeTimeFormat("en", {
-    unit: "best fit" // or "second", "minute", "hour", "day", "week", "month", "quarter" or "year" (defaults to "best fit")
+let rtf = new Intl.RelativeTimeFormat("en", {
+    style: "long" // "long" (default), "short", or "narrow"
 });
-console.log(a.format(Date.now() - 1000 * 60 * 60 * 25)); // yields "yesterday"
-console.log(a.format(Date.now() - 1000 * 60 * 60 * 50)); // yields "2 days ago"
 
-let b = new Intl.RelativeTimeFormat("en", {
-    unit: "hour"
-});
-console.log(b.format(Date.now() - 1000 * 60 * 60 * 25)); // yields "25 hours ago"
-console.log(b.format(Date.now() - 1000 * 60 * 60 * 50)); // yields "50 hours ago"
+rtf.format(
+  -1,
+  "day" // "second", "minute", "hour", "day", "week", "month", "quarter", or "year".
+);
+// > "1 day ago"
 
-let c = new Intl.RelativeTimeFormat("en", {
-    style: "narrow" // or "short" or "long" (defaults to "long")
-});
-console.log(c.format(Date.now() + 1000 * 60 * 60 * 24 * 30)); // yields "next mo."
-console.log(c.format(Date.now() + 1000 * 60 * 60 * 24 * 60)); // yields "in 2 mo."
+rtf.format(2.15, "day");
+// > "in 2 days"
+
+rtf.format(100, "day");
+// > "in 100 days"
+
+rtf.format(0, "day");
+// > "0 days ago" / "in 0 days" ??
```

Comment:
Note that we have this situation using this approach. Need to evaluate what ICU does...